crescent/errors.py

```diff
@@ -51,28 +54,23 @@ def decorator(callback: T | MetaStruct[T, Any]) -> MetaStruct[T, Any]:
 def catch_command(
     *exceptions: Type[Exception],
 ) -> Callable[
-    [CommandErrorHandlerCallbackT[Any] | MetaStruct[CommandErrorHandlerCallbackT[Any], Any]],
-    MetaStruct[CommandErrorHandlerCallbackT[Any], Any],
+    [CommandErrorHandlerCallbackT[Any]], MetaStruct[CommandErrorHandlerCallbackT[Any], Any],
 ]:
     return _catch_command(*exceptions)
 
 
 def catch_event(
     *exceptions: Type[Exception],
 ) -> Callable[
-    [EventErrorHandlerCallbackT[Any] | MetaStruct[EventErrorHandlerCallbackT[Any], Any]],
-    MetaStruct[EventErrorHandlerCallbackT[Any], Any],
+    [EventErrorHandlerCallbackT[Any]], MetaStruct[EventErrorHandlerCallbackT[Any], Any],
 ]:
     return _catch_event(*exceptions)
 
 
 def catch_autocomplete(
     *exceptions: Type[Exception],
 ) -> Callable[
-    [
-        AutocompleteErrorHandlerCallbackT[Any]
-        | MetaStruct[AutocompleteErrorHandlerCallbackT[Any], Any]
-    ],
+    [AutocompleteErrorHandlerCallbackT[Any]],
     MetaStruct[AutocompleteErrorHandlerCallbackT[Any], Any],
 ]:
     return _catch_autocomplete(*exceptions)
```

Comment:
You should just be able to declare these as variables. Pylance & mypy support that fine.

Reply:
I did them as functions party so your IDE will make them the function color and party cause typing the variables didn't work very well.

Reply:
hmm. Makes sense ig